com.alpineclient.AlpineClient.yml

```diff
@@ -0,0 +1,40 @@
+id: com.alpineclient.AlpineClient
+runtime: org.freedesktop.Platform
+runtime-version: "23.08"
+sdk: org.freedesktop.Sdk
+command: alpine-client
+finish-args:
+  - "--persist=.alpineclient"
+  - "--socket=x11"
+  - "--socket=pulseaudio"
+  - "--device=dri"
+  - "--share=network"
+  - "--share=ipc"
+  - "--filesystem=xdg-download:ro" # Drag & drop mods
+  - "--filesystem=xdg-run/app/com.spotify.Client:create" # Spotify integration
+  - "--filesystem=xdg-run/app/com.discordapp.Discord:create" # Discord integration
+modules:
+  - name: alpineclient
+    buildsystem: simple
+    build-commands:
+      - "install -D -m 755 alpine-client /app/bin/alpine-client"
+      - "install -D -m 644 com.alpineclient.AlpineClient.desktop /app/share/applications/com.alpineclient.AlpineClient.desktop"
+      - "install -D -m 644 com.alpineclient.AlpineClient.png /app/share/icons/hicolor/256x256/apps/com.alpineclient.AlpineClient.png"
+      - "install -D -m 644 com.alpineclient.AlpineClient.metainfo.xml /app/share/metainfo/com.alpineclient.AlpineClient.metainfo.xml"
+    sources:
+      - type: file
+        url: "https://github.com/alpine-client/pinnacle/releases/download/1.6.0/pinnacle-linux-amd64-ncgo"
+        sha256: "df15106b56cefc91a7daf61313e36c97ee149e17ef7199673dfbbeb76a8a45f6"
+        dest-filename: "alpine-client"
+      - type: file
+        path: "com.alpineclient.AlpineClient.desktop"
+      - type: file
+        path: "com.alpineclient.AlpineClient.png"
+      - type: file
+        path: "com.alpineclient.AlpineClient.metainfo.xml"
+
+  - name: xrandr
```

Comment:
this should be build before the main module since it's a dependency. that's the logicial order.

Reply:
I will fix this.